File: Practice12.c
#include<stdio.h>
#include<stdbool.h>
// Check range in 20 to 60 
bool CheckRange(int No1)
{
    if(((No1 >= 20)) && (No1 <= 60))
    {
        return true;
    }
    else
    {
        return false;
    }

}
int main()
{
    int iValue = 0;
    bool bRet = false;
    printf("Please enter number :\n");
    scanf("%d", &iValue);

    bRet = CheckRange(iValue);

    if(bRet == true)
    {
        printf("%d The number is in range \n", iValue);
    }
    else
    {
        printf("%d The number is not in range\n", iValue);
    }

    return 0;
}

File: Practice12.py
def CheckRange(No1):
    if 40 <= No1 and 70 >= No1:
        return 1
    else:
        return 2


def main():
    print("Plaease Enter Your marks")
    Num = int(input())
    Name = 0

    Name = CheckRange(Num)

    if (Name == 1):
        print("The Number is in range")
    else:
        print("The Number is not in range ")


if __name__ == "__main__":
    main()